### PR TITLE
Miscellaneous fixes

### PR DESCRIPTION
**Pull request by pypingou**

### CODE REVIEW

**Comment by laxathom** [comment]
:+1:  from me.
The utc thing works from a scratch fas3's db and a fas2's migration my side. Need to validate this on staging though.

**Comment by pypingou** [comment]
The amount of changes has increased quite a bit there ^^

File: fas/api/people.py [hunk 1]
```diff
@@ -16,21 +16,20 @@
 # along with this program; if not, write to the Free Software
 # Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
 #
-__author__ = ['Xavier Lamien <laxathom@fedoraproject.org>',
-              'Pierre-Yves Chibon <pingou@fedoraproject.org>']
+# __author__ = ['Xavier Lamien <laxathom@fedoraproject.org>',
+#              'Pierre-Yves Chibon <pingou@fedoraproject.org>']
```
Comment (pypingou) [inline fas/api/people.py]: If we comment it out, should we drop the `__author__` then?

**Comment by pypingou** [comment]
One remark, otherwise looks sane

**Comment by skrzepto** [comment]
login still doesnt work. try logging into fasadmin, the first time it works, the second time it fails with the same error as i reported.
 
I think the `UTCDateTime` implementation was actually correct

**Comment by laxathom** [comment]
hm...lemme guess, you're using an SQLite engine, right? 
I know that SQLite has a lot of caveat on datetime and sqlalchemy provides its own datetime impl for SQLite datetime formatting.

**Comment by skrzepto** [comment]
i think your right. we should then remove sqlitte from the *.ini files so no one accidently comes across these issues and force postgres as the dev db

link to src http://docs.sqlalchemy.org/en/latest/core/type_basics.html#sqlalchemy.types.DateTime.__init__

**Comment by skrzepto** [comment]
Just confirmed this works in postgresql. If we just make a note in the readme about using postgres only then I'm okay with this PR.

**Comment by skrzepto** [comment]
@laxathom @pypingou 

Are we deciding to stick with postgres for dev and prod db? This will solve the tz aware issue that is caused by sqlite.

**Comment by pypingou** [comment]
I think we should stick to postgres yes

@laxathom thoughts?

**Comment by laxathom** [comment]
:+1:  here.
However, let's not close the door to other DBAPI if people come around with a way to support them (MySQL, Oracle and the like)

**Comment by skrzepto** [comment]
@laxathom 
I think I read somewhere that MySQL has the same issues with storing timezone aware as sqlite. Thoughts? We can look at https://pypi.python.org/pypi/SQLAlchemy-UTCDateTime/  just not sure if we want to do that or not.

**Comment by laxathom** [comment]
@skrzepto let's open a discussion ticket to check this so we can move forward, merge and close this PR

**Comment by skrzepto** [comment]
lgtm as well

**Comment by pypingou** [comment]
Let's merge then :)

**Pull request merged**